main.go

```diff
@@ -25,22 +25,17 @@ import (
 var cpuprofile = flag.String("cpuprofile", "", "write cpu profile to file")
 
 func main() {
+	runtime.GOMAXPROCS(runtime.NumCPU())
```

Comment:
Adding this for Go backwards compatibility

Reply:
I don't think this program runs on Golang version <= 1.10, actually. We used a JSON API that is fairly new in the `encoding/json` https://golang.org/pkg/encoding/json/#Valida and due to a weird compiling error in CGO on Macs, it won't compile on anything less than 1.9 https://github.com/nccgroup/tracy/wiki/Building#golang. So not sure that changes anything here.

Reply:
https://github.com/nccgroup/tracy/blob/863645e99b0811751aa7aa1c78d4f0ffd8517f79/api/common/event.go#L251